settings.php

```diff
@@ -249,4 +254,8 @@
 
     $settings->add(new admin_setting_configcheckbox('tool_objectfs/preferexternal',
         new lang_string('settings:preferexternal', 'tool_objectfs'), '', ''));
+        
+    $settings->add(new admin_settings_aws_region('tool_objectfs/s3_region',
```

Comment:
Why is this setting necessary?

Reply:
This also was already in the code. Not sure..

Reply:
It should not appear in the diff. I think you need to rebase your changes.